test/fixtures/no-smart-oauth-uri-capability-statement.json

```diff
@@ -0,0 +1,235 @@
+{
+  "resourceType": "CapabilityStatement",
+  "status": "active",
+  "date": "2018-03-30T16:04:37+00:00",
+  "publisher": "Not provided",
+  "kind": "instance",
+  "software": {
+    "name": "HAPI FHIR Server",
+    "version": "2.4"
+  },
+  "implementation": {
+    "description": "HSPC Reference API Server"
+  },
+  "fhirVersion": "3.0.1",
+  "acceptUnknown": "extensions",
+  "format": [
+    "application/fhir+xml",
+    "application/fhir+json"
+  ],
+  "rest": [
+    {
+      "mode": "server",
+      "security": {
```

Comment:
In this case there wouldn't be a `security` key whatsoever, see: https://fhirtest.uhn.ca/conformance?serverId=home_21&pretty=false